dev/com.ibm.websphere.appserver.jcacheContainer-1.1/resources/l10n/com.ibm.websphere.appserver.jcacheContainer-1.1.properties

```diff
@@ -14,4 +14,6 @@
 #NLS_MESSAGEFORMAT_NONE
 #
 
-description=TODO write useful description for this feature before we beta it
+description=This feature enables support for applications that use the Java Caching API 1.1 \
```

Comment:
The first sentence is a bit misleading how it says "enables support for ... JCache".  The first and second sentence could be combined into a single sentence: "This feature provides Java Caching API 1.1 specification interfaces to applications."

Reply:
I'll get the description updated.